packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -82,14 +85,44 @@ async function main() {
     governorV2.address,
     existingVoting.address,
     votingV2.address,
+    proposer.address,
     finder.address,
     ownableContractsToMigrate,
     multicallContractsToMigrate
   );
 
   console.log("Deployed VotingUpgrader: ", votingUpgrader.address);
 
+  console.log("5. Deploying ProposerV2");
+
+  const defaultBond = hre.web3.utils.toWei("5000", "ether");
+
+  const proposerFactory: ProposerV2Ethers__factory = await getContractFactory("ProposerV2");
+  const proposerV2 = await proposerFactory.deploy(votingToken.address, defaultBond, governorV2.address, finder.address);
+  console.log("Deployed ProposerV2: ", proposerV2.address);
+
+  console.log("6. Set ProposerV2 as the proposer of GovernorV2");
+  await governorV2.resetMember(1, proposerV2.address);
+
+  console.log("7. Set the old governor as the owner of the new governor");
+  // The new governor owner will be updated in the VotingUpgraderV2 contract.
+  await governorV2.resetMember(0, governor.address);
+
+  console.log("8. Set the new governor as the owner of the new voting v2");
+  await votingV2.transferOwnership(governorV2.address);
+
   console.log("Deployment done!🎉");
+
+  console.log("Next step, Propose migration: ");
+  console.log(
+    `
+  ☝️ PROPOSAL: Run the following command to propose the migration to VotingV2, GovernorV2 and ProposerV2:
+  ${VOTING_UPGRADER_ADDRESS}=${votingUpgrader.address} \\
+  ${NEW_CONTRACTS.voting}=${votingV2.address} \\
+  ${NEW_CONTRACTS.governor}=${governorV2.address} \\
+  ${NEW_CONTRACTS.proposer}=${proposerV2.address} \\
+  yarn hardhat run ./src/upgrade-tests/voting2/1_Propose.ts --network ${hre.network.name}`.replace(/  +/g, "")
+  );
 }
 
 main().then(
```

Comment:
you'll extend this in a following PR for the additional functionality for the needed contracts, right?

Reply:
Yes, that was my plan!